x-pack/plugin/build.gradle

```diff
@@ -142,6 +142,19 @@ tasks.named("yamlRestCompatTestTransform").configure({ task ->
   task.skipTest("ml/sparse_vector_search/Search on a sparse_vector field with dots in the field names", "Vectors are no longer returned by default")
   task.skipTest("ml/sparse_vector_search/Search on a nested sparse_vector field with dots in the field names and conflicting child fields", "Vectors are no longer returned by default")
   task.skipTest("esql/190_lookup_join/lookup-no-key-only-key", "Requires the fix")
+  task.skipTest("esql/40_tsdb/aggregate_metric_double unsortable", "Extra function required to enable the field type")
```

Comment:
Hm, if we don't rename the now skipped tests, does that mean that the fwc test will be skipped for them forever? Would it be safer to update all the names of the changed yaml tests, e.g. by adding a `v2` suffix?

Reply:
Oh shoot. Uh. I don't know.

It won't be needed after I fix it to use the version. We can just un-skip it. I think.

But I'm not sure.